packages/revocation/src/contracts/RevocationRegistry.sol

```diff
@@ -0,0 +1,31 @@
+pragma solidity >=0.7.0 <0.9.0;
+
+/**
+* A contract for tracking the revocations of off-chain verifiable credentials.
+* For on-chain role revocations, see contract in iam-contracts repository.
+*/  
+contract RevocationRegistry {
```

Comment:
Not exactly the right place for this comment, but I think that it is strange that `RevocationRegistryOffChain.json` is committed under `constants`. I think it would be better to compile the contract from the solidity as needed (see `proxyIdentity` package for example)